From 4baf8e9f48f727d7d6a99adaf82e3e48e88d8679 Mon Sep 17 00:00:00 2001
From: Potin Lai <potin.lai@quantatw.com>
Date: Wed, 15 Dec 2021 10:35:42 +0800
Subject: [PATCH] Configurations: Bletchley: Update AspeedPwmFan

add FAN0 - FAN3

this commit relies on dbus-sensors change (49253)
https://gerrit.openbmc-project.xyz/c/openbmc/dbus-sensors/+/49253

Signed-off-by: Potin Lai <potin.lai@quantatw.com>
Change-Id: I933957549339f3b143ad0f7979f809e5a58887ef
---
 configurations/Bletchley.json | 224 ++++++++++++++++++++++++++++++++++
 1 file changed, 224 insertions(+)

diff --git a/configurations/Bletchley.json b/configurations/Bletchley.json
index 6f50dd5555..a5937cbeec 100644
--- a/configurations/Bletchley.json
+++ b/configurations/Bletchley.json
@@ -380,6 +380,230 @@
             "Name": "P12V_AUX_HSC",
             "Type": "ADM1278"
         },
+        {
+            "Connector": {
+                "Name": "System Fan Connector 0",
+                "Pwm": 0,
+                "Tachs": [
+                    0,
+                    1
+                ]
+            },
+            "Index": 0,
+            "PowerState": "Always",
+            "Name": "FAN0_TACH_IL",
+            "Thresholds": [
+                {
+                    "Direction": "greater than",
+                    "Name": "upper critical",
+                    "Severity": 1,
+                    "Value": 13000
+                },
+                {
+                    "Direction": "less than",
+                    "Name": "lower critical",
+                    "Severity": 1,
+                    "Value": 1000
+                }
+            ],
+            "Type": "AspeedPWMFan"
+        },
+        {
+            "Connector": {
+                "Name": "System Fan Connector 0",
+                "Pwm": 0,
+                "Tachs": [
+                    0,
+                    1
+                ]
+            },
+            "Index": 1,
+            "PowerState": "Always",
+            "Name": "FAN0_TACH_OL",
+            "Thresholds": [
+                {
+                    "Direction": "greater than",
+                    "Name": "upper critical",
+                    "Severity": 1,
+                    "Value": 13000
+                },
+                {
+                    "Direction": "less than",
+                    "Name": "lower critical",
+                    "Severity": 1,
+                    "Value": 1000
+                }
+            ],
+            "Type": "AspeedPWMFan"
+        },
+        {
+            "Connector": {
+                "Name": "System Fan Connector 1",
+                "Pwm": 1,
+                "Tachs": [
+                    2,
+                    3
+                ]
+            },
+            "Index": 2,
+            "PowerState": "Always",
+            "Name": "FAN1_TACH_IL",
+            "Thresholds": [
+                {
+                    "Direction": "greater than",
+                    "Name": "upper critical",
+                    "Severity": 1,
+                    "Value": 13000
+                },
+                {
+                    "Direction": "less than",
+                    "Name": "lower critical",
+                    "Severity": 1,
+                    "Value": 1000
+                }
+            ],
+            "Type": "AspeedPWMFan"
+        },
+        {
+            "Connector": {
+                "Name": "System Fan Connector 1",
+                "Pwm": 1,
+                "Tachs": [
+                    2,
+                    3
+                ]
+            },
+            "Index": 3,
+            "PowerState": "Always",
+            "Name": "FAN1_TACH_OL",
+            "Thresholds": [
+                {
+                    "Direction": "greater than",
+                    "Name": "upper critical",
+                    "Severity": 1,
+                    "Value": 13000
+                },
+                {
+                    "Direction": "less than",
+                    "Name": "lower critical",
+                    "Severity": 1,
+                    "Value": 1000
+                }
+            ],
+            "Type": "AspeedPWMFan"
+        },
+        {
+            "Connector": {
+                "Name": "System Fan Connector 2",
+                "Pwm": 2,
+                "Tachs": [
+                    4,
+                    5
+                ]
+            },
+            "Index": 4,
+            "PowerState": "Always",
+            "Name": "FAN2_TACH_IL",
+            "Thresholds": [
+                {
+                    "Direction": "greater than",
+                    "Name": "upper critical",
+                    "Severity": 1,
+                    "Value": 13000
+                },
+                {
+                    "Direction": "less than",
+                    "Name": "lower critical",
+                    "Severity": 1,
+                    "Value": 1000
+                }
+            ],
+            "Type": "AspeedPWMFan"
+        },
+        {
+            "Connector": {
+                "Name": "System Fan Connector 2",
+                "Pwm": 2,
+                "Tachs": [
+                    4,
+                    5
+                ]
+            },
+            "Index": 5,
+            "PowerState": "Always",
+            "Name": "FAN2_TACH_OL",
+            "Thresholds": [
+                {
+                    "Direction": "greater than",
+                    "Name": "upper critical",
+                    "Severity": 1,
+                    "Value": 13000
+                },
+                {
+                    "Direction": "less than",
+                    "Name": "lower critical",
+                    "Severity": 1,
+                    "Value": 1000
+                }
+            ],
+            "Type": "AspeedPWMFan"
+        },
+        {
+            "Connector": {
+                "Name": "System Fan Connector 3",
+                "Pwm": 3,
+                "Tachs": [
+                    6,
+                    7
+                ]
+            },
+            "Index": 6,
+            "PowerState": "Always",
+            "Name": "FAN3_TACH_IL",
+            "Thresholds": [
+                {
+                    "Direction": "greater than",
+                    "Name": "upper critical",
+                    "Severity": 1,
+                    "Value": 13000
+                },
+                {
+                    "Direction": "less than",
+                    "Name": "lower critical",
+                    "Severity": 1,
+                    "Value": 1000
+                }
+            ],
+            "Type": "AspeedPWMFan"
+        },
+        {
+            "Connector": {
+                "Name": "System Fan Connector 3",
+                "Pwm": 3,
+                "Tachs": [
+                    6,
+                    7
+                ]
+            },
+            "Index": 7,
+            "PowerState": "Always",
+            "Name": "FAN3_TACH_OL",
+            "Thresholds": [
+                {
+                    "Direction": "greater than",
+                    "Name": "upper critical",
+                    "Severity": 1,
+                    "Value": 13000
+                },
+                {
+                    "Direction": "less than",
+                    "Name": "lower critical",
+                    "Severity": 1,
+                    "Value": 1000
+                }
+            ],
+            "Type": "AspeedPWMFan"
+        },
         {
             "Address": "0x4F",
             "Bus": 0,
-- 
2.32.0

